examples/api/lib/widgets/interactive_viewer/interactive_viewer.builder.0.dart

```diff
@@ -38,71 +37,31 @@ class _IVBuilderExampleState extends State<_IVBuilderExample> {
   static const double _cellWidth = 200.0;
   static const double _cellHeight = 26.0;
 
-  // Returns true iff the given cell is currently visible. Caches viewport
-  // calculations.
-  Quad? _cachedViewport;
-  late int _firstVisibleRow;
-  late int _firstVisibleColumn;
-  late int _lastVisibleRow;
-  late int _lastVisibleColumn;
-  bool _isCellVisible(int row, int column, Quad viewport) {
-    if (viewport != _cachedViewport) {
-      final Rect aabb = _axisAlignedBoundingBox(viewport);
-      _cachedViewport = viewport;
-      _firstVisibleRow = (aabb.top / _cellHeight).floor();
-      _firstVisibleColumn = (aabb.left / _cellWidth).floor();
-      _lastVisibleRow = (aabb.bottom / _cellHeight).floor();
-      _lastVisibleColumn = (aabb.right / _cellWidth).floor();
-    }
-    return row >= _firstVisibleRow &&
-        row <= _lastVisibleRow &&
-        column >= _firstVisibleColumn &&
-        column <= _lastVisibleColumn;
-  }
-
-  // Returns the axis aligned bounding box for the given Quad, which might not
-  // be axis aligned.
-  Rect _axisAlignedBoundingBox(Quad quad) {
-    double? xMin;
-    double? xMax;
-    double? yMin;
-    double? yMax;
+  // Returns the axis aligned bounding box for the given Quad, which might not be axis aligned.
+  Rect axisAlignedBoundingBox(Quad quad) {
+    double xMin = quad.point0.x;
+    double xMax = quad.point0.x;
+    double yMin = quad.point0.y;
+    double yMax = quad.point0.y;
     for (final Vector3 point in <Vector3>[
-      quad.point0,
       quad.point1,
       quad.point2,
       quad.point3
     ]) {
-      if (xMin == null || point.x < xMin) {
+      if (point.x < xMin) {
         xMin = point.x;
-      }
-      if (xMax == null || point.x > xMax) {
+      } else if (point.x > xMax) {
         xMax = point.x;
       }
-      if (yMin == null || point.y < yMin) {
+
+      if (point.y < yMin) {
         yMin = point.y;
-      }
-      if (yMax == null || point.y > yMax) {
+      } else if (point.y > yMax) {
         yMax = point.y;
       }
     }
-    return Rect.fromLTRB(xMin!, yMin!, xMax!, yMax!);
-  }
 
-  void _onChangeTransformation() {
```

Comment:
Not sure why this was here - didn't seem to do anything

Reply:
Good catch.  I think I had another demo with the ability to tap a button to modify the TransformationController, and this code accidentally got left in here.